iot/spec/iot_spec.rb

```diff
@@ -0,0 +1,439 @@
+# Copyright 2018 Google, Inc
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+require "google/apis/cloudiot_v1"
+require "google/cloud/pubsub"
+require "rspec"
+require 'securerandom'
```

Comment:
Double Quotes everywhere `"`

*(Sorry we don't have a linter at the moment to pick these up! and some of the other syntax I'll call out)*

Reply:
Thanks for the suggested changes!